src/components/GetArticle.jsx

```diff
@@ -29,7 +33,11 @@ export const GetArticle = () => {
       <div className="idVotes">Votes: {article.votes}</div>
       <div className="idBody">{article.body}</div>
       <div className="idCreatedAt">Date: {article.created_at.slice(0, 10)}</div>
-      <div className="comments">Comments here</div>
+      <div className="comments">
+        <GetComments />
+        <AddVotes currentVotes={article.votes} article_id= {article_id} />
```

Comment:
You're already rendering Votes higher up on the page. This component should probably take it's place and not render votes twice